src/win/util.c

```diff
@@ -1380,6 +1380,7 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   }
 
   pwd->shell = NULL;
+  pwd->gecos = NULL;
```

Comment:
Sure, that makes sense. I think the same applies to `pwd->shell` so I did it this way just for consistency.